### ISSUE TITLE
Improve color contrast for accessibility

### ISSUE THREAD

**Issue by HaSistrunk** [sarapis/orservices#69]
## Is your feature request related to a problem? Please describe.
Some elements on the site lack sufficient color contrast between their color and the background color. On the dev instance, I'm seeing this in the: 
- navbar header in the contrast between the links and navbar color
- The footer between the white text and orange background
- The gray (`color: #9195a2`) text used for services and orgs description and information against its white background
- In many of the link `background-color` choices for the Service Category and Service Eligibility links
- Links in pagination
- The `<th>` text against its background color in the tables

## Describe the solution you'd like
Adjust those colors to meet the WCAG color contrast criteria. This specifies that text and images of text have a contrast ratio of at least 4:5:1. The ratio can be 3:1 if the text is large. Large means at least 18 point (pt not px) or 14 point bold. 

## Additional context
There are [a lot of nice tools](https://uxpro.cc/toolbox/accessibility/color-accessibility/) that will automatically check color contrast or test color combos for accessibility. Running automated checks on the site using a tool like [WAVE](https://wave.webaim.org/), [SiteImprove](https://siteimprove.com/en-us/core-platform/integrations/browser-extensions/), or [Accessibility Insights for Web](https://accessibilityinsights.io/docs/en/web/overview/) can help id this and other issues.

This accessibility guideline is specified in [WCAG 2.x Level AA (SC 1.4.3)](https://www.w3.org/WAI/WCAG21/quickref/#contrast-minimum).

**Comment by d9it** [comment]
The menu color & the footer color is dynamic from admin.

**Comment by d9it** [comment]
Change color contrast from the admin side. Below is the link 
https://nimb.ws/HXyN8r
Let me know if you have any doubts.

**Comment by HaSistrunk** [comment]
Thanks @d9it! Good call and thanks for the link. There are just a few things that don't seem to be dynamic from admin that I'm seeing with contrast issues. I've described below with some screenshots if that's useful:

1. The gray (`color: #9195a2`) text used for services and orgs description and information against its white background. An example is the description text for this resource:

![image](https://user-images.githubusercontent.com/15254041/131416631-93bdd140-870d-4e84-b45c-9698a47f3bf1.png)

3. The gray footer text at the very bottom of the page (`rgba(27, 27, 27, 0.4`):
![image](https://user-images.githubusercontent.com/15254041/131417144-61766a68-b309-418c-aa01-24aa2042bf9b.png)





2. Many of the background colors for the Service Category and Service Eligibility links. For example: Housing, Homeless, Men/Boys, Ages 18+, DC Resident, Care, Legal, Immigrants, Advocacy and Legal Aid... etc. all can be tough to read. This is a nice resource to generate a lot of color-safe colors if it's useful:[http://colorsafe.co/]( http://colorsafe.co/)
![image](https://user-images.githubusercontent.com/15254041/131415539-d03005d0-bc00-45d2-b574-18297e3bcfeb.png)
![image](https://user-images.githubusercontent.com/15254041/131416720-17f09318-9345-44bf-87c8-2a6ef8e84c1b.png)
![image](https://user-images.githubusercontent.com/15254041/131416754-6a3d4be2-ac9e-48b1-9143-07408aef9cb0.png)

**Comment by d9it** [comment]
The button background is managed from the admin side.
![Screenshot](https://user-images.githubusercontent.com/40555846/133210623-a5b3b3df-db8c-4df5-bf6e-32b403924230.png)
